### PR TITLE
Blog(e-commerce): Images

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
+---
+title: "Beautiful, efficient images for e‑commerce"
+author: samdutton
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
authors:
  - samdutton
```

Comment (robdodson) [reply src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]: https://github.com/GoogleChrome/web.dev/pull/856 switches up the syntax a bit so we can have more than one author on a post.

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+
++   **Use WebP with a JPEG fallback for product photos (not PNG!)** These
+    formats give you the [best compression and
+    highest quality](https://web.dev/serve-images-webp) for continuous tone
+    (raster) images such as photographs. WebP images are around
+    [25% smaller than JPEGs](https://developers.google.com/speed/webp/docs/webp_study)
+    of the same quality.
+
+    [WebP is supported by Chrome, Firefox and other browsers.](https://caniuse.com/#feat=webp)
+    You can easily [code for a JPEG fallback](web.dev/serve-images-webp) or use
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    You can easily [code for a JPEG fallback](/serve-images-webp) or use
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+    an image service such as [Cloudinary](https://cloudinary.com/) or
+    [Thumbor](https://github.com/thumbor/thumbor) that automatically delivers
+    WebP or JPEG depending on browser capability — without changing the file
+    extension. You can easily produce both WebP and JPEG versions of product
+    photos using [image automation in your build
+    workflow](https://web.dev/serve-images-webp), via [image
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    workflow](/serve-images-webp), via [image
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+    deliver WebP images to browsers that support the format instead of JPEGs —
+    without changing the file extension.
+
+    <br>
+
++   **Incorporate image processing in your website build process**
+
+    As part of your build process, you add steps to resize and compress images.
+    To find out more, see the guides at
+    [web.dev/fast/use-imagemin-to-compress-images](https://web.dev/fast/use-imagemin-to-compress-images).
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    [web.dev/use-imagemin-to-compress-images](/use-imagemin-to-compress-images).
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+    automate 'smart cropping' for different image sizes and automatically
+    deliver WebP images to browsers that support the format instead of JPEGs —
+    without changing the file extension.
+
+    <br>
+
++   **Incorporate image processing in your website build process**
+
+    As part of your build process, you add steps to resize and compress images.
+    To find out more, see the guides at
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    To find out more, see the guide at
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 6]
```diff
@@ -0,0 +1,404 @@
...
+
+    ``` html
+    <img
+      src="notebooks-small.jpg"
+      srcset="notebooks-medium.jpg 500w, notebooks-large.jpg 1000w"
+      alt="Notebooks">
+      ```
+
+    Find out more in our guide at
+    [web.dev/fast/serve-responsive-images](https://web.dev/fast/serve-responsive-images).
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    [web.dev/serve-responsive-images](/serve-responsive-images).
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 7]
```diff
@@ -0,0 +1,404 @@
...
+    high-DPI screens, and that purchase decisions may be made zooming in on an
+    image on a desktop machine. As well as testing your site on low-end mobile
+    devices, try it out on large and high quality displays. Be especially
+    careful of zoomable images (see below).
+
++   Consider using art direction for product images, providing different
+    crops for different display sizes. Take a look at the examples
+    [here](https://simpl.info/pictureart/) and
+    [here](http://udacity.github.io/responsive-images/examples/3-08/pictureArtDirection/),
+    and try out our codelab: [web.dev/fast/serve-responsive-images/codelab-art-direction](https://web.dev/fast/serve-responsive-images/codelab-art-direction).
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    and try out our codelab: [web.dev/codelab-art-direction](/codelab-art-direction).
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 8]
```diff
@@ -0,0 +1,404 @@
...
+    Network panel from your browser DevTools as you scroll the page.)
+
++   JavaScript libraries such as
+    [lazyload](https://github.com/verlok/lazyload),
+    [lozad](http://apoorv.pro/lozad.js/) and
+    [lazysizes](https://github.com/aFarkas/lazysizes): these use the
+    IntersectionObserver with fallbacks.
+
+Find out more from our guide
+[web.dev/fast/use-lazysizes-to-lazyload-images](https://web.dev/fast/use-lazysizes-to-lazyload-images).
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
[web.dev/use-lazysizes-to-lazyload-images](/use-lazysizes-to-lazyload-images).
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 9]
```diff
@@ -0,0 +1,404 @@
...
+    `img` element. This doesn't need to be difficult. You're likely to have
+    short product descriptions in your product database, and you can automate
+    the process of using these for `alt` attributes. Find out more about
+    writing good alt text from
+    [webaim.org/techniques/alttext](https://webaim.org/techniques/alttext/).
++   Make sure to provide text alternatives to graphical page elements such
+    as shopping carts and search icons.
+
+Learn how to improve accessibility from our guides at
+[web.dev/accessible](https://web.dev/accessible).
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
[web.dev/accessible](/accessible).
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 10]
```diff
@@ -0,0 +1,404 @@
...
+    the
+    [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report/).
++   Consider adding warnings to your build process, for example if
+    developers attempt to commit oversized files.
++   Sounds obvious, but if you're monitoring image usage on your live site,
+    make sure to test category and product pages — not just the home page!
+
+## Next steps
+
++   [web.dev/fast](https://web.dev/fast#topic-Optimize-your-images) has a
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
+   [web.dev/fast](/fast#optimize-your-images) has a
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 11]
```diff
@@ -0,0 +1,404 @@
...
+    range of guides and codelabs to help you optimize your images.
++   Our codelab
+    [Optimize images for e-commerce](https://docs.google.com/document/d/1m7xWYKf7gJ4IeLkKKc7FfhFtk0vZi0BM2nxJhzDPwHU/edit#heading=h.cgvdu9aeouaf)
+    [TODO: web.dev link] shows how to create beautiful images for online
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]: Missing TODO

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 12]
```diff
@@ -0,0 +1,404 @@
...
+
+How can you efficiently deliver images that drive engagement and conversion
+across a variety of devices, platforms and connectivity?
+
+{% Aside %}
+In this guide you'll find out what makes e-commerce different when it comes
+to images.
+
+You can learn more about general image optimization techniques at
+[web.dev/fast](https://web.dev/fast).
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
[web.dev/fast](/fast).
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md [hunk 13]
```diff
@@ -0,0 +1,404 @@
...
+
+### Hero images: give them individual attention
+
+You're likely to have a few especially important images that feature on your
+homepage or elsewhere. You need to especially careful to optimize these while
+maintaining quality.
+
++   **Use WebP with a JPEG fallback for product photos (not PNG!)** These
+    formats give you the [best compression and
+    highest quality](https://web.dev/serve-images-webp) for continuous tone
```
Comment (robdodson) [inline src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]:
```suggestion
    highest quality](/serve-images-webp) for continuous tone
```

File: src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md (hunk 11 above)
Comment (samdutton) [reply src/site/_drafts/beautiful-efficient-images-for-e-commerce/index.md]: I'll removed this temporarily, until the codelab is done.

**Comment by robdodson** [comment]
Per today's chat, I'm going to close out this PR

**Pull request closed without merging**